adding count()

AbbyJanke opened the pull request:
For #29 

So after further inspection It appears by using the Laravel Messenger back, `has()` already exists and works even with providing a specific level to check for. Unfortunately the built in `count()` only allows you to get total number that are in the MessengerBag not at a 'level' specific count. So I overwrote the original one which will allow you to get the total number of alerts only using the 'levels' specified in the config. But also allows you to enter a specific level such as 'success' to count.

Example Usage:
```php
$alertBag = [
   'success' = [
      'account updated',
      'new password saved',
   ],
   'danger' = [
      'update failed,
   ],
]

Alert::count(); // Will return total count of 3

Alert::count('danger'); // Will return total count under `danger` level, which is 1.

```

tabacitu commented:
Looks pretty much perfect to me 😄 I think this perfectly covers all use cases:
- ```Alert::has()``` & ```Alert::count()``` - see if there are any alerts
- ```Alert::has('error')``` & ```Alert::count('error')``` - see if there are any errors
- ```Alert::count('error')``` - see how many errors there are

Great! Thanks a lot @AbbyJanke , merging now.

tabacitu commented:
Wait. @AbbyJanke this should be non-breaking, right? This still works the way it used to, the way it's documented in the [displaying alerts](https://github.com/prologuephp/alerts#displaying-alerts) section. Right? 

How has ```has()``` changed  as opposed to the one provided by the MessageBag - just that it now accepts no parameter right?

AbbyJanke commented:
I ended up not touching `has()` whatsoever it worked right out of the box. I was just not realizing that was an option to start with as i never dug into the code for Alerts. I only modified the `count()` function to allow for a `key` to be added.

tabacitu commented:
Hmm... are you sure? Cause I can see a new ```has()``` method in the PR - https://github.com/prologuephp/alerts/pull/31/files#diff-8f34f23555aa744183ce5bddd2894e1aR106-R125

AbbyJanke commented:
@tabacitu dang, thought i had removed it but yes I had tested it before with that as well. The only thing that i had also added was to be certain that it was using the Alert message bag. xD my bad. Other then that I barely changed anything.

tabacitu commented:
Ok so you think it's ok to merge with this ```has()``` too, right?

AbbyJanke commented:
So  double reading the code, the`isEmpty()` function basically just runs the `any()` function. which literally checks is there anything within the messenger bag. I did that  in my code using `is_null()` and `isset()` instead of custom functions. 100% positive.

AbbyJanke commented:
Taylors code is easier to read, but I'm avoiding strenuous calls custom functions and using built in ones from PHP.

tabacitu commented:
Yeah no, I prefer legibility too 😄 Great - merged and tagged as 0.4.7. Thanks again @AbbyJanke !
